tests/fixtures/apps/demultiplexing/flow_cells_demux_all/211101_A00187_0615_AHLG5GDRXY/SampleSheet.csv

```diff
@@ -1,5 +1,5 @@
 [Data]
-FCID,Lane,SampleID,SampleRef,index,index2,SampleName,Control,Recipe,Operator,Project
```

Comment:
The flow cell seems to have been sequenced on a novaseqx machine but the structure of the sample sheet seems to be for bcl2fastq (?) Changed it to match the structure for bclconvert.

Reply:
This caused a test to fail when the pydantic model was migrated, not sure why it does not fail in master. I don't think it is a case we support currently anyway (running bcl2fastq on novaseqx)